docs/controls/controls.rst

```diff
@@ -0,0 +1,180 @@
+===================
+Qt Quick Controls 2
+===================
+
+.. sectionauthor:: `e8johan <https://github.com/e8johan>`_
+
+.. github:: ch105
+
+.. note::
+
+    Last Build: |today|
+
+    The source code for this chapter can be found in the `assets folder <../../assets>`_.
+
+This chapter introduces how to use Qt Quick Controls 2 to create a user interface built from standard components such as buttons, labels, sliders and so on. We will look at how various styles can be used to move user interfaces between platforms before diving into custom styling.
+    
+Introduction to Controls
+========================
+
+Using Qt Quick from scratch gives you primited graphical and interaction elements from which you can build your user interfaces. Using Qt Quick Controls 2 you start from a slightly more structured set of controls to build from. The controls range from simple text labels and buttons to more complex ones such as sliders and dials. These element are handy if you want to create a user interface based on classic interaction patterns as they provide a foundation to stand on.
+
+The Qt Quick Controls 2 comes with a number of styles out of the box that are shown in the table below. The *Default* style is a basic flat style. The *Universal* style is based on the Microsoft Universal Design Guidelines, while *Material* is based on Google's Material Design Guidelines, and the *Fusion* style is a desktop oriented style. 
+
+Some of the styles can be tweaked by tweaking the used palette. The *Imagine* is a style based on image assets, this allows a graphical designer to create a new style without writing any code at all, not even for palette colour codes.
+
+========= =====================================
+Style     Example
+========= =====================================
+Default   .. image:: assets/style-default.png
+Universal .. image:: assets/style-universal.png
+Material  .. image:: assets/style-material.png
+Fusion    .. image:: assets/style-fusion.png
+Imagine   .. image:: assets/style-imagine.png
+========= =====================================
+
+.. todo:: Pick a better UI to grab the example screenshot from.
+
+The Qt Quick Controls 2 is available from the ``QtQuick.Controls`` import module. In this module you will find the basic controls such as buttons, labels, checkboxes, sliders and so on. In addition to these controls, the following modules are also of interest:
+
+.. list-table::
+    :widths: 20 80
+    :header-rows: 1
+
+    *   - Module
+        - Description
+    *   - ``QtQuick.Controls``
+        - The basic controls.
+    *   - ``QtQuick.Layouts``
+        - Layout elements for supporting resizable user interfaces.
+    *   - ``QtQuick.Dialogs``
```

Comment:
This is a Controls 1 import, and although you can mix Controls 1 with Controls 2, I wouldn't put it in a page about Controls 2.

Reply:
Which one is? both layouts and dialogs, or only one?

Reply:
Dialogs (I always try to comment under the specific line).

Reply:
aha, I just removed Layouts - it also seems to be a 1.x import and I replaced the RowLayout used in the example with a Flow (as in the text editor example from the QML docs)

For dialogs, see the discussion below.